app/screens/Observations/map.js

```diff
@@ -100,8 +103,38 @@ class ObservationMapScreen extends Component {
     this.refs["menuInstance"].setVelocity({ x: 3000 });
   }
 
-  componentDidMount() {
-    console.log("map", this._map);
+  getFeatures() {
```

Comment:
You can skip the `.bind` on `:215` by writing this as:

```javascript
getFeatures = () => {
```

(pretty sure that RN is configured to take advantage of this feature)